etl_tests/test_assets.py

```diff
@@ -167,3 +174,35 @@ def test_wikipedia_anti_recommendations_json_file(
                 json.loads(wikipedia_json_line)[0][-1]
                 == anti_recommendation_graph[0][0][-1]
             )
+
+
+def test_wikipedia_arkg(
+    anti_recommendation_graph: tuple[
+        tuple[RecordKey, tuple[AntiRecommendationKey, ...]], ...
+    ],
+    arkg_config: ArkgConfig,
+    output_config: OutputConfig,
+    anti_recommendation_node_query: SparqlQuery,
+    anti_recommendation_key: AntiRecommendationKey,
+) -> None:
+    """Test that wikipedia_arkg successfully materializes a Wikipedia ARKG and writes it to file."""
+    wikipedia_arkg(
```

Comment:
You'll have the descriptor here.

Reply:
Yes, I am using the descriptor here now.